tools/src/test/java/org/apache/rocketmq/tools/command/broker/QueryMsgByIdSubCommandTest.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.rocketmq.tools.command.broker;
```

Comment:
1、please fork from  develop branch!
2、this case can imitate this pr：https://github.com/apache/rocketmq/pull/983